### PR TITLE
fix generator-langium:cli for esm

### PR DESCRIPTION
**Pull request by cdietrich**

### CODE REVIEW

**Comment by msujew** [comment]
@cdietrich You can run `npm link` in the `generator-langium` project and then run `yo langium`. That will automatically use the local package.

**Comment by cdietrich** [comment]
yes that worked for me

File: packages/generator-langium/templates/cli/bin/cli.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env node
 
-import '../out/cli/main';
+import main from '../out/cli/main.js';
+main();
```
Comment (msujew) [inline packages/generator-langium/templates/cli/bin/cli.js:4]: Minor: Please at a new line to the end of this file.

**Review by msujew — APPROVED** [review]
Thanks, works well for me 👍

File: packages/generator-langium/templates/cli/bin/cli.js (hunk 1 above)
Comment (cdietrich) [reply packages/generator-langium/templates/cli/bin/cli.js]: @msujew updated the PR

**Pull request merged**